src/main/java/com/ssbb/kanban/controller/project/ProjectController.java

```diff
@@ -26,9 +24,25 @@
 
 	@RequestMapping(value = "project", method = RequestMethod.GET)
 	public String loadProject(ModelMap map, HttpSession session) {
+
 		return "project";
 	}
 
+	@RequestMapping(value = "loadSelectedProject", method = RequestMethod.POST)
+	public String loadSelectedProject(ModelMap map, HttpSession session,
+			Project submittedProject) {
+
+		User user = (User) session.getAttribute(Constants.USER);
+		for (Project userProject : user.getProjects()) {

```

Comment:
@ilya123s Rather than looping through the entire list of projects, I think it would be better to write a new data call method in the ProjectDAO, getProjectByName(String name).

What do you think?